canonical-web-and-design/ubuntu.com
# Get the test backend flag from the URL

## Done

- Get the test backend flag from the URL.
- Remove old code for fetching the flag from the window.

## QA

- Visit [/advantage?test_backend=true](https://ubuntu-com-10331.demos.haus/advantage?test_backend=true) and log in
- Open your network dev tools.
- Check that there is a call to `/advantage/user-subscriptions?test_backend=true`.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/225.
